mlir/lib/Bindings/Python/Globals.cpp

```diff
@@ -6,25 +6,27 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "IRModule.h"
+#include "mlir/Bindings/Python/IRCore.h"
 
 #include <optional>
 #include <vector>
 
-#include "Globals.h"
-#include "NanobindUtils.h"
+#include "mlir/Bindings/Python/Globals.h"
+// clang-format off
```

Comment:
Any idea why we have a header ordering problem here (assuming clang-format off is to force a specific order)?